docs/DeveloperGuide.md

```diff
@@ -975,19 +916,95 @@ testers are expected to do more *exploratory* testing.
+This feature was originally deemed low priority, since a pair only has 2 fields: elderly and volunteer, and thus users can reasonably still achieve this by deleting a pair and adding a new pair. Through user feedback, we acknowledge that this is still a useful feature to have to improve efficiency and user experience.
+
+Proposed usage: Edits an existing pair based on their index in the pairs list.
+Proposed format: `edit_pair INDEX [eic/ELDERLY_NRIC] [vic/VOLUNTEER_NRIC]`
+
+Proposed behaviour:
+* Edits the pair at the specified `INDEX` in the displayed pair list.
+* Any combination of the optional fields is possible but **at least one** optional field must be specified.
+* Existing values will be updated to the input values.
+
+Examples:
+* `edit_pair 1 eic/T0245267I` Edits the 1st pair so that the volunteer is paired to the elderly with NRIC `T0245267I` instead.
+
+### Deleting a pair by index: `delete_pair`
+
+Currently, pairs are deleted by specifying NRIC of both elderly and volunteer. This feature can increase the efficiency of using FriendlyLink, if we allow users to simply specify the index of the pair.
+This feature is originally implemented as such to prevent accidental deletion of pairs, as it is easy to enter the wrong index but hard to accidentally enter a pair of incorrect NRICs and delete the wrong pair. Through user feedback, we acknowledge that we should implement it to delete by index and support this with an `undo` feature to minimise impact of accidental deletions.
+
+Proposed usage: Deletes an existing pair based on their index in the pairs list.
+Proposed format: `edit_pair INDEX [eic/ELDERLY_NRIC] [vic/VOLUNTEER_NRIC]`
+
+Proposed behaviour:
+* Edits the pair at the specified `INDEX` in the displayed pair list.
+* Any combination of the optional fields is possible but **at least one** optional field must be specified.
+* Existing values will be updated to the input values.
+
+### Display birthdate in person cards
+
+When users provide a birthdate for an Elderly or Volunteer profile, only the person's current age is displayed in the app. To view the specific birthdate, users must refer to the respective JSON file.
+We should display the birthdate in FriendlyLink as well.
+
+### Enhance display of large number of tags
+
+There are no limits on the number of tags or the character limit for each tag. If large number of tags or tags with excessively long names are used, it affects the UI display and impact user experience.
+<Insert proposed solution>
+
+### Enhance display of pair information
+
+Currently, users need to mouse over the pair card to view information of each pair. This reduces the efficiency of FriendlyLink, which is intended to be optimised for typing. A command to view pairing details, instead of hovering over the pair item, and reading from the popover card would improve efficiency.
+<Insert proposed solution>
```

Comment:
Currently, users need to mouse over the pair card to view information of each pair. This reduces the efficiency of FriendlyLink, which is intended to be optimised for typing. 

In the future, we plan to add a command to view pairing details directly as information cards. This would help to improve effeciency of display.